torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -35,6 +36,7 @@ static std::unordered_map<std::string, ParameterType> type_map = {
   {"ScalarType", ParameterType::SCALARTYPE},
   {"Layout", ParameterType::LAYOUT},
   {"MemoryFormat", ParameterType::MEMORY_FORMAT},
+  {"LinalgBackend", ParameterType::LINALG_BACKEND},
```

Comment:
This also wasn't done for setting context like deterministic flags. @ngimel suspects if you follow the deterministic flag PRs more closely you can avoid a lot of this